bin/Powheg/production/2017/13TeV/Higgs/gg_H_quark-mass-effects_JHUGen_AnomalousCouplings_NNPDF31_13TeV/gg_H_quark-mass-effects_NNPDF31_13TeV.input

```diff
@@ -0,0 +1,71 @@
+numevts NEVENTS
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1           ! hadron 2 (1 for protons, -1 for antiprotons)
+#ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+#ndns2 131         ! pdf set for hadron 2 (mlm numbering)
+ebeam1 6500     ! energy of beam 1
+ebeam2 6500    ! energy of beam 2
+
+lhans1 306000       ! pdf set for hadron 1 (LHA numbering)
+lhans2 306000       ! pdf set for hadron 2 (LHA numbering)
```

Comment:
could you please change the pdf to "325300"

Reply:
Ok I will change the pdf set here and then in the meantime i will make another PR to the branch you specified